src/codegen/c.zig

```diff
@@ -3348,6 +3348,31 @@ fn airBuiltinCall(f: *Function, inst: Air.Inst.Index, fn_name: [*:0]const u8) !C
     return local;
 }
 
+fn airCountZeroes(f: *Function, inst: Air.Inst.Index, which_zeroes: enum { leading, trailing }) !CValue {
+    if (f.liveness.isUnused(inst)) return CValue.none;
+
+    const inst_ty = f.air.typeOfIndex(inst);
+    const local = try f.allocLocal(inst_ty, .Const);
+    const operand = f.air.instructions.items(.data)[inst].ty_op.operand;
+    const operand_ty = f.air.typeOf(operand);
+    const target = f.object.dg.module.getTarget();
+    const writer = f.object.writer();
+
+    const fn_name = switch (which_zeroes) {
+        .leading => "zig_clz",
+        .trailing => "zig_ctz",
+    };
```

Comment:
You know what's even better than branching twice? Only branching once! Just pass `fn_name` into `airCountZeroes`:

```zig
     try writer.print(" = zig_{s}(", .{fn_name}); 
```

Reply:
Done.